src/binaryen-c.h

```diff
@@ -230,6 +229,7 @@ struct BinaryenLiteral {
     double f64;
     uint8_t v128[16];
     const char* func;
+    // std::unique_ptr exn;
```

Comment:
What's the plan here?

Reply:
The C-API currently can't represent `ExceptionPackage`s (for a generator to access), while the C++-API can. Would you prefer that I make a better comment here, or remove it for now?

Reply:
Removed the comment, can think about this later.